### ISSUE TITLE
Unable to login and publish extension via vsce and PAT

### ISSUE THREAD

**Issue by ljvmiranda921** [microsoft/vsmarketplace#259]
Hi everyone, 

I am trying to publish a VSCode extension via the `vsce` command-line tool. However, I always get an Access Denied error. Something in the likes of this:

```bash
$ vsce login Foo
https://marketplace.visualstudio.com/manage/publishers
Personal Access Token for publisher 'Foo': *************************
Error: Access Denied: xxxx-xxx-xxx-xxxx needs the following permission(s) on the resource /Foo to perform this action: View user permissions on a resource
```

It also happens whenever I try to publish using Github actions:

```bash
$ vsce publish --pat ${{ secrets.VSCE_TOKEN }}
ERROR  Access Denied: xxxx-xxx-xxx-xxxx needs the following permission(s) on the resource /Foo to perform this action: Publish new extensions to an existing publisher
```

I can confirm that:
- **The publisher ID is indeed `Foo` (title case).** I tried uploading the extension manually and it was successful. When I checked the complete address of the extension, it registers as `Foo.my-extension`
- **My PAT has the "Manage" scope** in the Marketplace
- **My PAT has "All accessible organizations"** set when creating it
- **The `publisher` key in my `package.json` has the value `Foo`**
- **I'm using the same account for creating the PAT and creating the Publisher**

I'd definitely prefer to have my extension be published from Github actions and I don't want to upload it manually every time there's a release. I've checked a lot of threads such as https://github.com/microsoft/vscode-vsce/issues/11 . https://github.com/microsoft/vscode-vsce/issues/507. etc. 

I even tried creating new PATs while carefully following this documentation [here](https://code.visualstudio.com/api/working-with-extensions/publishing-extension) + reading the [FAQs](https://code.visualstudio.com/api/working-with-extensions/publishing-extension#common-questions).

As a first-time VSCode extension developer, I'm definitely missing something very obvious here. Hope someone can point me in the right direction. The only difference is **that I didn't create my organization as per the docs. Instead, I created a publisher under an organization that I'm already part of.**

Thanks a lot!

```
$ vsce --version
2.5.0
$ node --version
v16.9.1
$ npm --version
8.1.4
```

**Comment by yannduran** [comment]
@ljvmiranda921 did you ever find out the solution to this annoying problem? I'm having the exact same problem trying to publish a Visual Studio extension via a Github action.

**Comment by stephenfuqua** [comment]
I was able to solve the exact error message above by granting Graph: read. But then I ran into another error, and was unable to resolve that even by granting the token Full Access permissions and using "all accessible organizations". Same Microsoft account.

**Comment by stephenfuqua** [comment]
The problem seems to occur [if you have multiple Microsoft accounts](https://github.com/Microsoft/vscode-vsce/issues/277#issuecomment-489584722). My account is tied in with SSO back to my employer's Active Directory. It _looks_ like I have only one account, but that might not be true, as I'm also a external user in an Azure Active Directory instance.

So I tried signing-in to the Marketplace using a "service agent" account that is setup only in Azure Active Directory. The PAT I created with that account worked on the first try, with only the "marketplace: manage" permission as documented.
